File: sdks/thunkmetrc-java/wrapper/src/main/java/io/github/thunkier/thunkmetrc/wrapper/models/Additive.java
package io.github.thunkier.thunkmetrc.wrapper.models;
import com.fasterxml.jackson.annotation.JsonProperty;public class Additive {
    @JsonProperty("AdditiveTypeName")
    public String additiveTypeName;
    @JsonProperty("AmountUnitOfMeasure")
    public String amountUnitOfMeasure;
    @JsonProperty("ApplicationDevice")
    public String applicationDevice;
    @JsonProperty("EpaRegistrationNumber")
    public String epaRegistrationNumber;
    @JsonProperty("Note")
    public String note;
    @JsonProperty("PlantBatchId")
    public Integer plantBatchId;
    @JsonProperty("PlantBatchName")
    public String plantBatchName;
    @JsonProperty("PlantCount")
    public Integer plantCount;
    @JsonProperty("ProductSupplier")
    public String productSupplier;
    @JsonProperty("ProductTradeName")
    public String productTradeName;
    @JsonProperty("Rate")
    public String rate;
    @JsonProperty("RestrictiveEntryIntervalQuantityDescription")
    public String restrictiveEntryIntervalQuantityDescription;
    @JsonProperty("RestrictiveEntryIntervalTimeDescription")
    public String restrictiveEntryIntervalTimeDescription;
    @JsonProperty("TotalAmountApplied")
    public Integer totalAmountApplied;
    @JsonProperty("Volume")
    public Double volume;
}

File: sdks/thunkmetrc-py/wrapper/src/thunkmetrc/wrapper/models/additive.py
from typing import TypedDict, List, Optional, Any, Generic, TypeVar, TYPE_CHECKING
from typing_extensions import TypedDict

if TYPE_CHECKING:
    from . import *

class Additive(TypedDict, total=False):
    AdditiveTypeName: str
    AmountUnitOfMeasure: str
    ApplicationDevice: str
    EpaRegistrationNumber: str
    Note: str
    PlantBatchId: int
    PlantBatchName: str
    PlantCount: int
    ProductSupplier: str
    ProductTradeName: str
    Rate: str
    RestrictiveEntryIntervalQuantityDescription: str
    RestrictiveEntryIntervalTimeDescription: str
    TotalAmountApplied: int
    Volume: float
